app/schemas/schema.js

```diff
@@ -1,112 +1,230 @@
 const schema = {
-  "title": "Request to Participate",
   "type": "object",
   "required": [
-    "organization",
-    "streetNumber",
-    "streetName",
-    "city",
-    "postalCode"
+    // "Organization",
+    // "Registration Number",
+    // "Street Number",
+    // "Street Name",
+    // "City",
+    // "Postal Code",
+    // "Primary Contact",
+    // "Postion",
+    // "Contact Email",
+    // "Telephone",
+    // "Zone Feedback",
+    // "Wired Broadband",
+    // "Fixed Wireless",
+    // "Last Mile Support",
+    // "Anticipated Projects",
+    // "Underserved Areas",
+    // "Satellite Proposal",
+    // "Capital Cost Funding",
+    // "Capital Cost Explanation",
+    // "Last Mile Broadband",
+    // "Last Mile Fixed Wireless",
+    // "Last Mile New Backbone",
+    // "Highway Cellular"
   ],
   "properties": {
-    "Applicant Profile:": {
+    "Organization Profile": {
+      "title": "Organization Profile",
+      "type": "object",
       "properties": {
-        "organization": {
-          "title": "Applicant organization (legal name):",
+        "Organization": {
+          "title": "Organization name (legal name)",
           "type": "string"
         },
-        "registrationNumber": {
+        "Registration Number": {
```

Comment:
Why change these from camelCase variable names to titles with spaces? These are the data keys that will be defined in our json object when the field is filled.

Reply:
Ah ok, when I wasn't explicit with the "title" prop it looked wonky. But I'll go back to camelCase